docker/build_scripts/install-entrypoint.sh

```diff
@@ -19,5 +20,9 @@ if [ "${AUDITWHEEL_PLAT}" == "manylinux2010_i686" ] || [ "${AUDITWHEEL_PLAT}" ==
 	# the following script takes care of cleaning-up some things
 	# and since it's also needed in the finalize step, everything's
 	# centralized in this script to avoid code duplication
-	LC_ALL=C ${MY_DIR}/update-system-packages.sh
+	LC_ALL=C "${MY_DIR}"/update-system-packages.sh
```

Comment:
It feels weird to see double quotes around `${MY_DIR}` only.
Again, I don't know much about raw POSIX shell.
If `${MY_DIR}/update-system-packages.sh` doesn't work for some reason, maybe `"${MY_DIR}/update-system-packages.sh"` does ?

Reply:
Shellcheck complains because unquoted variables containing spaces would get parsed as two strings. `"${MY_DIR}/update-system-packages.sh"` works just as well, so I changed it to that.